docs/README.md

```diff
@@ -13,6 +13,7 @@ This is the documentation of AMS, it's focused on the **0.10.x version.**
 
 - [How to add root key](howto/add_root_key.md)
 - [How to add pagination links](howto/add_pagination_links.md)
+- [How to add top-level links](howto/add_top_level_links.md) (```JSON-API``` only)
```

Comment:
or better yet, just one 'add_links.md' page since they're the same thing more or less